fireapp/lib/presentation/login/login_view_model.dart

```diff
@@ -64,4 +67,14 @@ class LoginViewModel
     password.dispose();
   }
 
+  void registerTokens() async {
+    var userID =
+        (await _authenticationRepository.getCurrentSession())?.userId;
+    if (userID == null) {
+      throw Exception(
+          'User ID is null. Cannot register FCM Tokens.');
+    }
+     await _fcmTokensRepository.registerFCMTokens(userID);
```

Comment:
An await means nothing if you don't await the parent function. Also something is funky with the whitespace, please figure out what code is indented incorrectly and fix it